scripts/install

```diff
@@ -126,6 +128,54 @@ gpasswd --add www-data $app
 #=================================================
 ynh_script_progression --message="Setting up source files..."
 
+if [ $(uname -m) == "armv7l" ] 
+then
+	# prepare openjdk8 installation
+	if  [ -d "/tmp/jitsi-install/" ]
+	then
+		rm -rf /tmp/jitsi-install/
+	fi
+	
+	mkdir /tmp/jitsi-install/
+	#install openjdk 8 because javah is used by jitsi and not supported in higher versions
+	wget http://security.debian.org/debian-security/pool/updates/main/o/openjdk-8/openjdk-8-jre_8u322-b06-1~deb9u1_armhf.deb -P /tmp/jitsi-install/
+	wget http://security.debian.org/debian-security/pool/updates/main/o/openjdk-8/openjdk-8-jre-headless_8u322-b06-1~deb9u1_armhf.deb -P /tmp/jitsi-install/
+	wget http://security.debian.org/debian-security/pool/updates/main/o/openjdk-8/openjdk-8-jdk_8u322-b06-1~deb9u1_armhf.deb -P /tmp/jitsi-install/
+	wget http://security.debian.org/debian-security/pool/updates/main/o/openjdk-8/openjdk-8-jdk-headless_8u322-b06-1~deb9u1_armhf.deb -P /tmp/jitsi-install/
+	
+	sudo apt install \
+	 /tmp/jitsi-install/openjdk-8-jre_8u322-b06-1~deb9u1_armhf.deb \
+	 /tmp/jitsi-install/openjdk-8-jre-headless_8u322-b06-1~deb9u1_armhf.deb \
+	 /tmp/jitsi-install/openjdk-8-jdk_8u322-b06-1~deb9u1_armhf.deb \
+	 /tmp/jitsi-install/openjdk-8-jdk-headless_8u322-b06-1~deb9u1_armhf.deb
```

Comment:
instead of all that stuff, maybe we can have some arm dependencies like at https://github.com/YunoHost-Apps/kodi_ynh/blob/master/scripts/_common.sh

Reply:
i will try it.
never used _common.sh.. all have a first time

Reply:
watching for that openjdk8 is not anymore in default debian10 apt source-list